src/main/java/Duke/tasks/Todo.java

```diff
@@ -0,0 +1,12 @@
+package Duke.tasks;
+
+import Duke.Task;
+
+public class Todo extends Task {
+    public Todo(String description){
```

Comment:
```suggestion
    public Todo(String description) {
```

Don't forget to leave a space between your brackets and braces!